src/Syntax/TimeAgnostic.hs

```diff
@@ -212,7 +228,8 @@ data Exp (a :: ActType) (t :: Timing) where
   ByStr :: Pn -> String -> Exp AByteStr t
   ByLit :: Pn -> ByteString -> Exp AByteStr t
   ByEnv :: Pn -> EthEnv -> Exp AByteStr t
-
+  -- contracts
+  Call   :: Pn -> SType a -> Id -> [TypedExp t] -> Exp a t
```

Comment:
I'm wondering if this should be `Create` instead of `Call`, since we use it only for contract creation, and we don't really have a notion of specifying actual calls within act right now?

Reply:
Yes, I think that's a great idea!

Reply:
Done!